global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "6.0.100-rc.1.21458.32"
+    "version": " 6.0.100-rc.1.21458.32 "
```

Comment:
Adding those spaces makes the version not valid. See https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x